examples/CMakeLists.txt

```diff
@@ -111,6 +116,11 @@ if (ENABLE_CUDA)
     multi_device
     PROPERTIES CUDA_SEPARABLE_COMPILATION On)
 
+  target_compile_options(
+    multi_device PUBLIC
```

Comment:
Did this not get inherited from the `umpire` target?

Reply:
Over did it.  Oops.  This has been removed (nice catch).